epick_driver/src/default_driver.cpp

```diff
@@ -171,14 +202,10 @@ void DefaultDriver::deactivate()
   request.push_back(data_utils::get_msb(crc));
   request.push_back(data_utils::get_lsb(crc));
 
-  try
-  {
-    serial_->write(request);
-    auto response = serial_->read(kDectivateResponseSize);
-  }
-  catch (const serial::IOException& e)
+  auto response = send(request, kDectivateResponseSize);
+  if (response.empty())
   {
-    RCLCPP_ERROR(kLogger, "Failed to deactivate the gripper: %s", e.what());
+    RCLCPP_ERROR(kLogger, "Failed to deactivate the gripper.");
     throw;
```

Comment:
Should this create the exception to be thrown now that it's no longer inside a catch block? Or maybe the intention was to not throw and handle the error in some other way?

Reply:
I added a custom DriverException.